app/js/components/manageApps/ManageApps.jsx

```diff
@@ -368,7 +376,9 @@ export default class ManageApps extends React.Component {
             </div>
           </div>
         </div>
-        <div className="waiting-modal"><p className="upload-text">Uploading {uploadStatus}%</p></div>
+        {deleteStatus ? 
+          <div className="waiting-modal"><p className="upload-text">Deleting Addon</p></div>
+          : <div className="waiting-modal"><p className="upload-text">Uploading {uploadStatus}%</p></div>}
```

Comment:
Isn't this redundant from the upload progress bar we have that shows upload percentage stat?

Reply:
@paulupendo, The `upload progress bar` was changed to an overlay in AOM-39, based on feedback we got from @suthagar23. The idea was to ensure the user doesn't have access to use other UI elements on the app when an upload is in progress, so as to prevent the user from breaking the add-on upload mid-cycle. This PR builds on it for when an add-on deletion is in progress.